examples/combined_ODNP.py

```diff
@@ -99,8 +97,8 @@
 powers = 1e-3 * 10 ** (dB_settings / 10.0)
 # }}}
 # {{{ these change if we change the way the data is saved
-IR_postproc = "spincore_IR_v1"  # note that you have changed the way the data is saved, and so this should change likewise!!!!
-Ep_postproc = "spincore_ODNP_v3"
+IR_postproc = "spincore_IR_v3"  # note that you have changed the way the data is saved, and so this should change likewise!!!!
+Ep_postproc = "spincore_ODNP_v5"
```

Comment:
increment post processing version so that we aren't multiplying by nScans anymore